recipes/bond/all/conanfile.py

```diff
@@ -0,0 +1,55 @@
+from conans import ConanFile, CMake, tools
+import platform, os
+
+
+class BondConan(ConanFile):
+    name = "bond"
+    version = "9.0.0"
+    license = "MIT License"
+    homepage = "https://github.com/microsoft/bond.git"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Bond is a cross-platform framework for working with schematized data. It supports cross-language de/serialization and powerful generic mechanisms for efficiently manipulating data. Bond is broadly used at Microsoft in high scale services."
+    topics = ("bond", "microsoft")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False]}
+    default_options = {
+        "fPIC": True}
+    build_requires = "boost/1.71.0"
+    generators = "cmake"
+    exports_sources = ["FindBond.cmake"]
+
+    def source(self):
+        self.run("git clone https://github.com/microsoft/bond.git")
+        self.run("git checkout fe6f582ce4beb65644d9338536066e07d80a0289", cwd='bond')
+        self.run("git submodule update --init --recursive", cwd='bond')
+        tools.replace_in_file("bond/CMakeLists.txt",
+                              "project (bond)",
+                              '''project (bond)
+message("BOOST_ROOT=$ENV{BOOST_ROOT}")''')
+
+    def configure_cmake(self):
+        if platform.system() == "Linux":
```

Comment:
```suggestion
    def _configure_cmake(self):
        if tools.os_info.is_linux:
```

You can't add public methods, use protected instead.